packages/tests/server/inference.test.ts

```diff
@@ -0,0 +1,244 @@
+import {
+  inferProcedureInput,
+  inferProcedureOutput,
+  initTRPC,
+} from '@trpc/server/src';
+import { Observable, observable } from '@trpc/server/src/observable';
+import { z } from 'zod';
+import { expectTypeOf } from './inferenceUtils';
```

Comment:
cc @mmkal how far is https://github.com/mmkal/expect-type/pull/16 from getting merged? 👀

Reply:
On leave for a few weeks so you did the right thing by merging. Will get to it by the end of the year. Hoping it won't take long but I'm AFK til I'm back!

Reply:
I'm hoping to take a look in the next week or so - since it is now April (I guess I was talking about the US tax year)

Reply:
Nearly one year on... https://github.com/mmkal/expect-type/pull/16 is _almost_ ready! But I see trpc now uses the export from vitest anyway 👍